test/lib/action_cable/subscription_adapter/solid_cable_test.rb

```diff
@@ -136,10 +136,18 @@ class ActionCable::SubscriptionAdapter::SolidCableTest < ActionCable::TestCase
     end
   end
 
+  test "silencing polling queries when there's no Active Record logger" do
+    with_active_record_logger(nil) do
+      @rx_adapter.send(:listener).send(:broadcast_messages)
+    end
+
+    assert true
+  end
+
   private
     def cable_config
       { adapter: "solid_cable", message_retention: "1.second",
-        polling_interval: "0.01.seconds" }
+        polling_interval: "0.01.seconds", silence_polling: true }
```

Comment:
It's the default value, but needed to add it there for my mocking method with_active_record_logger